rust_snuba/rust_arroyo/src/backends/kafka/config.rs

```diff
@@ -33,6 +34,20 @@ impl KafkaConfig {
             .config_map
             .insert("auto.offset.reset".to_string(), auto_offset_reset);
 
+        config.config_map.insert(
+            "max.poll.interval.ms".to_string(),
+            max_poll_interval_ms.to_string(),
+        );
+
+        // HACK: If the max poll interval is less than 45 seconds, set the session timeout
+        // to the same. (its default is 45 seconds and it must be <= to max.poll.interval.ms)
+        if max_poll_interval_ms < 45_000 {
+            config.config_map.insert(
+                "session.timeout.ms".to_string(),
+                max_poll_interval_ms.to_string(),
+            );
+        }
```

Comment:
Most of the consumer config items are resolved in Python for both the Python and Rust consumers to avoid duplication. How about moving this inside https://github.com/getsentry/snuba/blob/b928a73ed7764829ae9bd897f007468d12c3c791/snuba/consumers/consumer_config.py#L125 (which is passed to the rust consumer via `consumer_config_raw`).